script/Ciel/Day 7/07_01_CIEL07_5.txt

```diff
@@ -420,7 +420,7 @@ Vlov was... unrelated?
 [offset:15907]{
 -- Page 462, Offset 15907. Glued. Mods: @x, @n.
 -- ……
-...%{n}
+...
```

Comment:
![image](https://user-images.githubusercontent.com/90473809/196247087-781c0bb9-7346-4cab-ad7c-c0c0f477305e.png)

Reply:
Yeah I did this because we generally don't like starting the line with a lead-in ellipsis. Honestly you could probably just make this a `%{nothing}`